core/src/test/java/com/microsoft/applicationinsights/telemetry/JsonTelemetryDataSerializerTest.java

```diff
@@ -275,18 +282,27 @@ public void test() throws IOException {
         stubClass.getM1().put("key1", 5);
         stubClass.getM1().put("key2", 6);
         stubClass.getM1().put("key3", 7);
+        stubClass.setSeverity(SeverityLevel.Critical);
         StringWriter stringWriter = new StringWriter();
         JsonTelemetryDataSerializer tested = new JsonTelemetryDataSerializer(stringWriter);
         stubClass.serialize(tested);
         tested.close();
         String str = stringWriter.toString();
-        StubClass bac = new Gson().fromJson(str, StubClass.class);
+        System.out.println("serailized=\n"+str);
```

Comment:
Do we need this print statement here? I believe it will output on console which everyone (other devs) would not be aware of without looking at the code.

Reply:
Gradle suppresses sysout by default, but I can take this out.